modules/ROOT/pages/architecture.adoc

```diff
@@ -42,41 +42,6 @@ The minimum requirements are 3 controller and 3 worker nodes. 3 controllers enab
 
 Mule applications run on workers. Multiple replicas of applications can run across workers.
 
-== Scaling Considerations
```

Comment:
@paigehargrave Is this context moved somewhere else?

Reply:
Yes.